### DISCUSSION TITLE
allOf support

### DISCUSSION THREAD

**Discussion by jkone27** [fsprojects/SwaggerProvider#251] [Q&A]
i noticed many time in genration `obj` type is used instead of proper one in case of oneOf, maybe just the first json variant should be considered?  in my case even with just 1 case of allOf or oneOf the method would not show anything other than (obj) in the method signature

```yaml
notWorking:
          allOf:
            - $ref: '#/components/schemas/SomeType'
          nullable: true
working:
          $ref: '#/components/schemas/SomeType'
```

https://swagger.io/docs/specification/v3_0/data-models/oneof-anyof-allof-not/

**Comment by sergey-tihon** [comment]
Should be already fixed here https://github.com/fsprojects/SwaggerProvider/issues/279#issuecomment-3689897989
